inc/class-cachify.php

```diff
@@ -1392,7 +1392,7 @@ public static function set_cache( $data ) {
 		 * @param string $cache_hash    The cache hash.
 		 * @param int    $cache_expires Cache validity period.
 		 */
-		$should_cache = apply_filters( 'cachify_store_item', true, $data, self::$method, self::_cache_hash(), self::_cache_expires() );
+		$should_cache = apply_filters( 'cachify_store_item', http_response_code() === 200, $data, self::$method, self::_cache_hash(), self::_cache_expires() );
```

Comment:
[`http_response_code`](https://www.php.net/manual/function.http-response-code.php) was introduced in PHP 5.4

`Cachify` is yet compatible with 5.2

If we merge this for the next point or major release (likely 2.4), I'd vote for raising the minimum PHP version so something more reasonable above 5.4 and ignore this issue for now.

Reply:
Seems we already have a vote for this in #259

So let's do it.

Reply:
If we're talking about style, we should add some line breaks, too.